CHANGELOG.md

```diff
@@ -5,6 +5,192 @@ information on the list of deprecated flags and APIs please have a look at
++ Add `--force` to `docker service update` [#27596](https://github.com/docker/docker/pull/27596)
++ Add `-q` to `docker service ps` [#27654](https://github.com/docker/docker/pull/27654)
+* Display number of global services in `docker service ls` [#27710](https://github.com/docker/docker/pull/27710)
+- Remove `--name` flag from `docker service update`. This flag is only functional on `docker service create`, so was removed from the `update` command [#26988](https://github.com/docker/docker/pull/26988)
+- Fix worker nodes failing to recover because of transient networking issues [#26646](https://github.com/docker/docker/issues/26646)
+* Add support for health aware load balancing and DNS records [#27279](https://github.com/docker/docker/pull/27279)
+* Add `--hostname` to `docker service create` [#27857](https://github.com/docker/docker/pull/27857)
+- Add `--tty` flag to `docker service create`/`update` [#28076](https://github.com/docker/docker/pull/28076)
+* Autodetect, store, and expose node IP address as seen by the manager [#27910](https://github.com/docker/docker/pull/27910)
+* Encryption at rest of manager keys and raft data [#27967](https://github.com/docker/docker/pull/27967)
++ Add `--update-max-failure-ratio`, `--update-monitor` and `--rollback` flags to `docker service update` [#26421](https://github.com/docker/docker/pull/26421)
+- Fix an issue with address autodiscovery on `docker swarm init` running inside a container [#26457](https://github.com/docker/docker/pull/26457)
++ (experimental) Add `docker service logs` command to view logs for a service [#28089](https://github.com/docker/docker/pull/28089)
+- Pin images by digest for `docker service create` and `update` [#28173](https://github.com/docker/docker/pull/28173)
+- Add short (`-f`) flag for `docker node rm --force` and `docker swarm leave --force` [#28196](https://github.com/docker/docker/pull/28196)
++ Don't repull image if pinned by digest [#28265](https://github.com/docker/docker/pull/28265)
++ Windows server 2016 swarm-mode support [#27838](https://github.com/docker/docker/pull/27838)
+
+### Volume
+
++ Add support for labels on volumes [#25628](https://github.com/docker/docker/pull/21567)
++ Add support for filtering volumes by label [#25628](https://github.com/docker/docker/pull/25628)
+* Add a `--force` flag in `docker volume rm` to forcefully purge the data of the volume that has already been deleted [#23436](https://github.com/docker/docker/pull/23436)
+* Enhance `docker volume inspect` to show all options used when creating the volume [#26671](https://github.com/docker/docker/pull/26671)
+* Add support for local NFS volumes to resolve hostnames [#27329](https://github.com/docker/docker/pull/27329)
+
+### Security
+
+- Fix selinux labeling of volumes shared in a container [#23024](https://github.com/docker/docker/pull/23024)
+- Prohibit `/sys/firmware/**` from being accessed with apparmor [#26618](https://github.com/docker/docker/pull/26618)
+
+### DEPRECATION
```

Comment:
- Deprecate `MAINTAINER` in `Dockerfile` https://github.com/docker/docker/pull/25466